fitlins/cli/run.py

```diff
@@ -91,6 +91,8 @@ def get_parser():
                         help='regex pattern to include files')
     g_bids.add_argument('--exclude', action='store', default=None,
                         help='regex pattern to exclude files')
+    g_bids.add_argument('--desc', action='store', default='preproc',
```

Comment:
To be more consistent with `--participant-label`:

```suggestion
    g_bids.add_argument('--desc-label', action='store', default='preproc',
```